From 71a520a4c03bf1e7c2a767abd47f7bd72dc48f44 Mon Sep 17 00:00:00 2001
From: Emanuel Buholzer <emanuel0xb@gmail.com>
Date: Mon, 22 May 2023 21:43:24 +0000
Subject: [PATCH] Don't use backward_ros

---
 controller_manager/CMakeLists.txt | 1 -
 1 file changed, 1 deletion(-)

diff --git a/controller_manager/CMakeLists.txt b/controller_manager/CMakeLists.txt
index 1d9b062..404ff0b 100644
--- a/controller_manager/CMakeLists.txt
+++ b/controller_manager/CMakeLists.txt
@@ -19,7 +19,6 @@ set(THIS_PACKAGE_INCLUDE_DEPENDS
 find_package(ament_cmake REQUIRED)
 find_package(ament_cmake_python REQUIRED)
 find_package(ament_cmake_core REQUIRED)
-find_package(backward_ros REQUIRED)
 foreach(Dependency IN ITEMS ${THIS_PACKAGE_INCLUDE_DEPENDS})
   find_package(${Dependency} REQUIRED)
 endforeach()
--
2.40.1